### ISSUE TITLE
Discourse - Adsense plugin crashes discourse rebuild in the latest version

### ISSUE THREAD

**Issue by kazilotus** [discoursehosting/discourse-adsense#24]
During discourse upgrade there seemed to be an unexpected issue. Upon further reviewing the log it mention something about an invalid <br> tag. After that I tried to rebuild discourse from terminal, but not long after executing the rebuild command, it returned with an error about failing to install bootstrap. I then removed the git link from app.yml and ran the rebuild command again and then everything seemed to work as it should have. So I suppose there are some compatibility issues with this plugin.
I hope this issue will be fixed soon, this was a great plugin (until it started showing problems :P)

P.S If you require, I can recreate the issue and submit a screenshot.

**Comment by discoursehosting** [comment]
That is correct, we only support the 'stable' branch of Discourse, so the highest supported version is 1.2.4 (as can be seen in the README).

Discourse does not have any way to check plugin compatibility, and breaks plugins frequently because backwards compatibility is not being maintained on third party plugins.

When 1.3 is out of beta we will support it very soon.

**Comment by kazilotus** [comment]
params:
  version: stable

I added this in app.yml and included adsense plugin and rebuild the app, Voila! it worked!
